packages/dayjs/index.ts

```diff
@@ -1,16 +1,17 @@
 /* eslint-disable @calcom/eslint/deprecated-imports */
-import dayjs from "dayjs";
 import dayjsBusinessTime from "dayjs-business-days2";
-import customParseFormat from "dayjs/plugin/customParseFormat";
-import duration from "dayjs/plugin/duration";
-import isBetween from "dayjs/plugin/isBetween";
-import isToday from "dayjs/plugin/isToday";
-import localizedFormat from "dayjs/plugin/localizedFormat";
-import minmax from "dayjs/plugin/minMax";
-import relativeTime from "dayjs/plugin/relativeTime";
-import timeZone from "dayjs/plugin/timezone";
-import toArray from "dayjs/plugin/toArray";
-import utc from "dayjs/plugin/utc";
+
+import dayjs from "@calcom/dayjs-fork";
```

Comment:
Named this `@calcom/dayjs-fork` because inside the monorepo we have another package already called `@calcom/dayjs` which imports the `dayjs` package. Want to avoid collisions/confusion around this and didn't think it warranted going through the entire app to change all of the `@calcom/dayjs` imports to something like `@calcom/dayjs{something}`.